doc/sphinx/_static/coqdoc.css

```diff
@@ -9,6 +9,69 @@
 /************************************************************************/
 /* Taken from CoqDoc's default stylesheet */
 
+
+.id[title="constructor"],.coqdoc-constructor {
+    color: rgb(225 0 0);
+}
+
+.id[title="var"],.coqdoc-var,
+.id[title="variable"],.coqdoc-variable {
+    color: rgb(210 46 0);
+}
+
+.id[title="definition"],.coqdoc-definition {
+    color: #FF540A;
+}
+
+.id[title="abbreviation"],.coqdoc-abbreviation {
+    color: #0042b9;  font-weight: bold
+
+}
+
+.id[title="lemma"],.coqdoc-lemma {
+    color: rgb(255 85 0);
+  font-weight: bold
+}
+
+.id[title="instance"],.coqdoc-instance {
+    color: #003eae;  font-weight: bold
+
+}
+
+.id[title="projection"],.coqdoc-projection {
+    color: #0042b9;  font-weight: bold
+
+}
+
+.id[title="method"],.coqdoc-method {
+    color: #0d62fb;  font-weight: bold
+
+}
+
+.id[title="inductive"],.coqdoc-inductive {
+    color: rgb(0%,0%,80%);
+}
+
+.id[title="record"],.coqdoc-record {
+    color: rgb(0%,0%,80%);
+}
+
+.id[title="class"],.coqdoc-class {
+    color: rgb(200 105 8);
+}
+
+.id[title="keyword"],.coqdoc-keyword {
+ color: #08288d;
+ font-weight: bold
+}
+
+
+
+
+
+/*
```

Comment:
Did you intend to keep a commented out code block? Shouldn't it be removed?

Reply:
Indeed